resource-types/GithubStatus.dhall

```diff
@@ -0,0 +1,52 @@
+{ meta = { name = "github-status", repository = "resource/github-status" }
+, Version = { Type = {}, default = {=} }
```

Comment:
The version does have a type, it's ``{ `context@sha`: Text }``. See https://github.com/colstrom/concourse-github-status/blob/03c4e0e2273f6318a825328d43c7be269cdf4cf6/lib/github-status/check.rb#L34